cpu/cc26xx-cc13xx/rf-core/prop-mode.c

```diff
@@ -249,13 +264,24 @@ volatile static uint8_t *rx_read_entry;
 
 static uint8_t tx_buf[TX_BUF_HDR_LEN + TX_BUF_PAYLOAD_LEN] CC_ALIGN(4);
 /*---------------------------------------------------------------------------*/
+static void
+rf_check_rx_err()
+{
+  int ret;
```

Comment:
This unused variable generates a warning, which in turn causes travis to fail (we build with `-Werror`). Please fix.